plotting/timeseries.py

```diff
@@ -109,7 +109,9 @@ def load_data(self):
                 if factor != 1.0:
                     point_data[idx] = np.multiply(point_data[idx], factor)
 
-            times = dataset.timestamps[self.starttime:self.endtime + 1]
```

Comment:
This was using the netcdf timestamps as indices...which doesn't make sense, so all I had to do was convert these to time indices.